proposals/jib_core_library.md

```diff
@@ -31,46 +38,67 @@ Design for Jib Core as a Java library for building container images.
 Three `TargetImage` types (`RegistryImage`, `DockerDaemonImage`, and `TarImage`) define the 3 different targets Jib can build to.
 
 `RegistryImage` - builds to a container registry
+
 - `static RegistryImage named(ImageReference/String)`
 - `RegistryImage addCredential(String username, String password)`
 - `RegistryImage addCredentialRetriever(CredentialRetriever)`
 
 `DockerDaemonImage` - builds to a Docker daemon
+
 - `static DockerDaemonImage named(ImageReference/String)`
 - `DockerDaemonImage setDockerExecutable(Path)`
 
 `TarImage` - builds to a tarball archive
+
 - `Builder`
   - `TarImage saveTo(Path outputFile)`
 - `static Builder named(ImageReference/String)`
 
 `Containerizer` - configures how and where to containerize to
-- `static Containerizer to(RegistryImage)`
```

Comment:
I think the reasoning for having the static initializers on `Containerizer` is to increase discoverability since `JibContainerBuilder#containerize` takes a `Containerizer` - so it's like of like `Streams.of` and then `Stream#collect(Collector)` with `Collectors.to...`.

Reply:
The examples put these methods on Jib (`Jib.to`), not `Containerizer`. Will update the examples instead.

Reply:
Ah oops that was a mistake then. Thanks!